firebase_admin/auth.py

```diff
@@ -85,17 +85,27 @@ def verify_id_token(id_token, app=None):
     Args:
       id_token: A string of the encoded JWT.
       app: An App instance (optional).
+      check_revoked: check whether the token was revoked(optional).
 
     Returns:
       dict: A dictionary of key-value pairs parsed from the decoded JWT.
 
     Raises:
       ValueError: If the JWT was found to be invalid, or if the App was not
           initialized with a credentials.Certificate.
+      AuthError: If check_token is requested and the token was revoked.
     """
     token_generator = _get_auth_service(app).token_generator
-    return token_generator.verify_id_token(id_token)
-
+    verified_claims = token_generator.verify_id_token(id_token)
+    if check_revoked:
+        user = get_user(verified_claims.get('uid'), app)
+        if  verified_claims.get('iat') < user.tokens_valid_after_time:
+            raise AuthError('ID_TOKEN_REVOKED', 'The Firebase ID token has been revoked.')
+    return verified_claims
+
+def revoke_refresh_tokens(uid, app=None):
+    user_manager = _get_auth_service(app).user_manager
+    user_manager.update_user(uid, tokens_valid_after_time=str(int(time.time())))
```

Comment:
Unnecessary wrapping in a str IMO

Reply:
It works without it, but this conforms to the other timestamps (the golang json tags also wrap this field in quotes.)